client/app/services/ownership-service-modal/ownership-service-modal.component.js

```diff
@@ -78,7 +77,7 @@ function ComponentController ($state, lodash, CollectionsApi, EventNotifications
   }
 
   // Private
-  function activate () {
+  vm.$onInit = function () {
```

Comment:
Can we hoist this lifecycle hook to the top of the component to better align with every other component out there?

Reply:
Yeah, i thought about doing that but decided not to shift too much within the file.  I can move it though.